app/src/main/java/com/cornellappdev/android/eatery/onboarding/OnboardingLoginFragment.java

```diff
@@ -0,0 +1,33 @@
+package com.cornellappdev.android.eatery.onboarding;
```

Comment:
do you plan on making this different than our current `AccountInfoFragment`? Is there a reason to not reuse the `AccountInfoFragment`?

Reply:
![image](https://user-images.githubusercontent.com/43295248/68783321-cbfbc000-0608-11ea-8284-ba4b5d74e5cc.png)
![image](https://user-images.githubusercontent.com/43295248/68783333-d1f1a100-0608-11ea-98c3-1a65749cc9b1.png)
The two are physically different, so I thought it'd make sense to separate them. But, they do share most of the same functionality. @connorreinhold Let me know what you guys think.

Reply:
Yeah - separate designs so I'd say we should have a separate fragment for each. A lot of the login functionality has been abstracted out to LoginUtilities in Network, so we can reuse the code there

Reply:
<img src="https://user-images.githubusercontent.com/43295248/68783920-be930580-0609-11ea-98ea-8a6accb0fbb0.png" width="24%"> <img src="https://user-images.githubusercontent.com/43295248/68783948-c652aa00-0609-11ea-9979-be055ceb8c28.png" width="24%">
The two are physically different, so I thought it'd make sense to separate them. Let me know what you guys think @connorreinhold